code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -446,3 +446,47 @@
 
 /obj/item/clothing/suit/hospital/pink
 	color = "#ffb7db"
+
+/obj/item/clothing/suit/letterman/Initialize()
```

Comment:
Put these procs after the type definition on line 466, but before the subtype definitions beginning on 474